custom_components/tryfi/pytryfi/fiPet.py

```diff
@@ -160,6 +190,93 @@ def updateAllDetails(self, sessionId: requests.Session):
         # TODO: Support weekly
         self._dailySleep, self._dailyNap = self._extractSleep(petJson['dailySleepStat'])
         self._monthlySleep, self._monthlyNap = self._extractSleep(petJson['monthlySleepStat'])
+        # Try to fetch behavior data for Series 3+ collars
+        try:
+            self.updateBehaviorStats(sessionId)
+        except Exception:
+            # Behavior stats may not be available for older collars
+            pass
+
+    # Update behavior stats for Series 3+ collars
+    def updateBehaviorStats(self, sessionId: requests.Session):
+        """Update behavior statistics for Series 3+ collars."""
+        try:
+            healthTrendsJSON = query.getPetHealthTrends(sessionId, self.petId, 'DAY')
+            behavior_trends = healthTrendsJSON.get('behaviorTrends', [])
+            self.setBehaviorStatsFromTrends(behavior_trends)
+            return True
+        except Exception as e:
+            LOGGER.debug(f"Could not update behavior stats for Pet {self.name}. This may be an older collar model.\n{e}")
+            return False
+    
+    def setBehaviorStatsFromTrends(self, behaviorTrends):
+        """Parse behavior data from health trends API."""
+        # Reset all metrics
+        self._dailyBarkingCount = 0
+        self._dailyBarkingDuration = 0
+        self._dailyLickingCount = 0
+        self._dailyLickingDuration = 0
+        self._dailyScratchingCount = 0
+        self._dailyScratchingDuration = 0
+        self._dailyEatingCount = 0
+        self._dailyEatingDuration = 0
+        self._dailyDrinkingCount = 0
+        self._dailyDrinkingDuration = 0
+        
+        try:
+            for trend in behaviorTrends:
+                if not isinstance(trend, dict):
+                    continue
+                    
+                trend_id = trend.get('id', '')
+                summary = trend.get('summaryComponents', {})
+                
+                # Extract events count
+                events_summary = summary.get('eventsSummary')
+                events_count = 0
+                if events_summary and 'event' in events_summary:
+                    try:
+                        events_count = int(events_summary.split()[0])
+                    except Exception:
+                        pass
+                
+                # Extract duration
+                duration_summary = summary.get('durationSummary')
+                duration_seconds = 0
+                if duration_summary:
+                    try:
+                        if duration_summary.startswith('<'):
+                            duration_seconds = 30
+                        else:
+                            parts = duration_summary.replace('h', '').replace('m', '').split()
+                            if len(parts) == 2:
+                                duration_seconds = int(parts[0]) * 3600 + int(parts[1]) * 60
```

Comment:
I don't have a sample response, so I'm not quite sure what the durationSummary looks like which makes this block of code kind of confusing. Is it like 1h3m or a different format?